packages/roosterjs-color-utils/lib/utils/getDarkColor.ts

```diff
@@ -0,0 +1,21 @@
+import * as Color from 'color';
+
+const baseLValue = 21.247; // L-value of #333333
+
+/**
+ * Get dark mode color for a given color
+ * @param color The color to calculate from
+ */
+export default function getDarkColor(color: string): string {
+    try {
+        const computedColor = Color(color || undefined);
+        const colorLab = computedColor.lab().array();
+        const newLValue = (100 - colorLab[0]) * ((100 - baseLValue) / 100) + baseLValue;
```

Comment:
Not sure if this is the standard way to get the dark mode color?

Reply:
This is copied from OWA